<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Components_Manager_Ui_DockWidget</class>
 <widget class="QDockWidget" name="Components_Manager_Ui_DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Components Manager</string>
  </property>
  <widget class="QWidget" name="Components_Manager_Ui_dockWidgetContents">
   <layout class="QGridLayout" name="Components_Manager_Ui_dockWidgetContents_gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="Components_Manager_Ui_splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="Components_Manager_Ui_treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="Components_Informations_groupBox">
       <property name="title">
        <string>Components Informations</string>
       </property>
       <layout class="QGridLayout" name="Components_Informations_groupBox_gridLayout">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="Components_Informations_textBrowser">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
